db/file_indexer.h

```diff
@@ -58,7 +58,8 @@ class FileIndexer {
                    std::vector<FileMetaData*>* const files);
 
   enum {
-    kLevelMaxIndex = std::numeric_limits<int32_t>::max()
+    // MSVC version 1800 still does not have constexpr for ::max()
+    kLevelMaxIndex = INT32_MAX
```

Comment:
@siying INT32_MAX is not in limits.h and may not be in stdint.h so it is fare. Do you want a similar approach for other constants or just this one?